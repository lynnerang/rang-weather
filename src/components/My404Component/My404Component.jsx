import React from 'react';

const My404Component = () => {
  return (
    <div>
      <h2>404 - Page not found.</h2>
    </div>
  );
};

export default My404Component;