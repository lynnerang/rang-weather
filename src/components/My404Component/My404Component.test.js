import React from 'react';
import { shallow } from 'enzyme';
import My404Component from './My404Component';

describe('My404Component', () => {
  const wrapper = shallow(<My404Component />);

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})