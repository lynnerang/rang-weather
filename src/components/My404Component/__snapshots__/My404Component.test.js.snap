// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My404Component matches the snapshot 1`] = `
<div>
  <h2>
    404 - Page not found.
  </h2>
</div>
`;
